jenkins/Jenkinsfile-blossom.premerge

```diff
@@ -191,6 +191,12 @@ pipeline {
                 if (currentBuild.currentResult == "SUCCESS") {
                     githubHelper.updateCommitStatus("$BUILD_URL", "Success", GitHubCommitState.SUCCESS)
                 } else {
+                    // upload log only in case of build failure
+                    def guardWords = ["gitlab.*?\\.com", "urm.*?\\.com"]
+                    def logPattern = "### BEGIN OF TEST LOG ###.*### END OF TEST LOG ###"
+
+                    githubHelper.uploadPartialLogs(this, env.JOB_NAME, env.BUILD_NUMBER, null, guardWords, logPattern)
```

Comment:
where can I find the docs for uploadPartialLogs?

Reply:
replied the link to you via email